lighthouse-core/audits/dobetterweb/bloated-libraries.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+/**
+ * @fileoverview This audit checks a page for any unnecessarily large npm dependencies.
+ * These "bloated" libraries can be replaced with functionally equivalent, smaller ones.
+ */
+
+'use strict';
+
+const Audit = require('../audit.js');
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of a Lighthouse audit that provides detail on large Javascript libraries that are used on the page that have better alternatives. This descriptive title is shown when to users when no known "bloated" libraries are detected on the page.*/
+  title: 'Avoids unnecessarily large JavaScript dependencies',
+  /** Title of a Lighthouse audit that provides detail on large Javascript libraries that are used on the page that have better alternatives. This descriptive title is shown when to users when some known "bloated" libraries are detected on the page.*/
+  failureTitle: 'Includes unnecessarily large JavaScript dependencies',
+  /** Description of a Lighthouse audit that tells the user why they should care about the large Javascript libraries that have better alternatives.. This is displayed after a user expands the section to see more. No character length limits. */
+  description: 'Large JavaScript libraries can lead to poor performance. ' +
+    'Prefer smaller, functionally equivalent libraries to reduce your bundle size.' +
+    '[Learn more](https://developers.google.com/web/fundamentals/performance/webpack/decrease-frontend-size#optimize_dependencies',
+  /** Label for a column in a data table. Entries will be names of suggested libraries that are an alternative to what's currently being used. */
+  suggestion: 'Suggestion',
+};
+
+const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
+const database = {'moment': 'dayjs'};
+
+class BloatedLibrariesAudit extends Audit {
+  /**
+   * @return {LH.Audit.Meta}
+   */
+  static get meta() {
+    return {
+      id: 'bloated-libraries',
+      title: str_(UIStrings.title),
```

Comment:
```suggestion
      title: str_(UIStrings.title),
      failureTitle: str_(UIStrings.failureTitle),
```

need to add the failure title here too :)